Tests/New-ProGetFeed.Tests.ps1

```diff
@@ -0,0 +1,115 @@
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Tests.ps1' -Resolve)
+
+function Initialize-ProGetFeedTests
+{
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)]
+        [object]
+        $ProGetSession
+    )
+    
+    $Global:Error.Clear()
+    
+    # Remove all feeds from target ProGet instance
+    Invoke-PGNativeApiMethod -Session $ProGetSession -Name 'Feeds_GetFeeds' -Parameter @{IncludeInactive_Indicator = $true} |
+        ForEach-Object { 
+            Invoke-PGNativeApiMethod -Session $ProGetSession -Name 'Feeds_DeleteFeed' -Parameter @{Feed_Id = $PSItem.Feed_Id}
+        }
+
+
+}
+
+Describe 'New-ProGetFeed.create a new Universal package feed' {
+    
+    $session = New-ProGetTestSession
+    Initialize-ProGetFeedTests -ProGetSession $session
+
+    New-ProGetFeed -ProGetSession $session -FeedType 'ProGet' -FeedName 'Apps'
+    $feedExists = Get-ProGetFeed -ProGetSession $session -FeedType 'ProGet' -FeedName 'Apps'
+    
+    It 'should write no errors' {
+        $Global:Error | Should BeNullOrEmpty
+    }
+    
+    It 'should create a new ProGet universal feed' {
+        $feedExists | Should Be $true
+    }
+}
+
+Describe 'New-ProGetFeed.when attempting to create a duplicate package feed' {

```

Comment:
Duplicate this test and add an `-ErrorAction Ignore` parameter to the second call to `New-ProGetFeed`. Test that no errors are written.